central/globaldb/close.go

```diff
@@ -10,4 +10,7 @@ func Close() {
 	if rocksDB != nil {
 		rocksDB.Close()
 	}
+	if postgresDB != nil {
+		postgresDB.Close()
```

Comment:
I was reading and had the concern that there is the potential that this call could block until the inflight queries are complete.  Should we send a deadline context the the overarching function so we don't wait forever?  I think it would be a rare problem, but it is possible:
https://github.com/jackc/pgx/issues/802
https://stackoverflow.com/questions/71588402/how-to-force-closing-pgxpool-pool-while-queries-are-still-running

Reply:
@dashrews78 So based on my read of those two links, basically just having the application die is the method in the worst case which is currently set to 30s by k8s

Reply:
We don't have access to a global context for all running contexts so there is no way to send a cancel

Reply:
right.  I wasn't aware of the 30s k8s setting.  That should be more than sufficient.